services/tests/test_models.py

```diff
@@ -122,10 +120,10 @@ def test_download_gdpr_data_returns_empty_dict_if_request_fails(
 def test_remove_service_gdpr_data_no_url(profile, service):
     service_connection = ServiceConnectionFactory(profile=profile, service=service)
 
-    with pytest.raises(MissingGDPRUrlException):
-        service_connection.delete_gdpr_data(api_token="token", dry_run=True)
-    with pytest.raises(MissingGDPRUrlException):
-        service_connection.delete_gdpr_data(api_token="token")
+    result = service_connection.delete_gdpr_data(api_token="token", dry_run=True)
```

Comment:
Some changes in `test_models.py` are now in the last commit (bb539d2). But that commit doesn't change `services/models.py`. That file is only changed in commit 042db232 (which also includes _some_ changes to `test_models.py`). Should all changes to `test_models.py` also be in that same commit?

Reply:
I just changed the existing tests to work with the style of error reporting implemented in the last commit.

The tests wouldn't work if they are changed before the implementation changes?

Reply:
The last commit bb539d28 only changes `profiles/schema.py` from the production code files (rest of the changes are in test files). If changes in `profiles/schema.py` affect tests in `services/tests/test_models.py`, something is not in the correct place or there are dependencies between modules that shouldn't probably be there.

Reply:
Ah. My mistake, I had committed the tests in the wrong commit.